compiler/modules/front/stmt.bal

```diff
@@ -422,8 +438,9 @@ function codeGenForeachStmt(StmtContext cx, bir:BasicBlock startBlock, Environme
 
 function codeGenWhileStmt(StmtContext cx, bir:BasicBlock startBlock, Environment env, s:WhileStmt stmt) returns CodeGenError|StmtEffect {
     bir:BasicBlock loopHead = cx.createBasicBlock(); // where we go to on continue
-    bir:BranchInsn branchToLoopHead = { dest: loopHead.label, pos: stmt.body.startPos };
-    startBlock.insns.push(branchToLoopHead);
+    cx.openRegion(loopHead.label, "LOOP");
```

Comment:
You should use the enum constants you have defined rather than strings.

Reply:
Using enum constants there gives an undefined error